packages/eslint-plugin-turbo/lib/utils/getEnvVarDependencies.ts

```diff
@@ -21,39 +20,56 @@ function findDependsOnEnvVars({
 
 function getEnvVarDependencies({
   cwd,
-  turboConfig,
 }: {
   cwd: string | undefined;
-  turboConfig?: Schema;
-}): Set<string> | null {
-  const turboJsonContent = turboConfig || findTurboConfig({ cwd });
-  if (!turboJsonContent) {
+}): Record<string, Set<string>> | null {
+  const turboConfigs = getTurboConfigs(cwd);
+
+  if (!turboConfigs.length) {
     return null;
   }
-  const {
-    globalDependencies,
-    globalEnv = [],
-    pipeline = {},
-  } = turboJsonContent;
-
-  const allEnvVars: Array<string> = [
-    ...findDependsOnEnvVars({
-      dependencies: globalDependencies,
-    }),
-    ...globalEnv,
-  ];
-
-  Object.values(pipeline).forEach(({ env, dependsOn }) => {
-    if (dependsOn) {
-      allEnvVars.push(...findDependsOnEnvVars({ dependencies: dependsOn }));
+
+  const envVars: Record<string, Set<string>> = {
+    "//": new Set(),
```

Comment:
I'm guessing `//` means root here also, might be good to add a comment or a named constant